pkgs/applications/editors/emacs/elisp-packages/yes-no/default.nix

```
@@ -0,0 +1,26 @@
+{ lib, fetchurl, trivialBuild }:
+
+trivialBuild {
+  pname = "yes-no";
+  version = "0";
+
+  src = fetchurl {
+    url = "https://github.com/emacsmirror/emacswiki.org/blob/185fdc34fb1e02b43759ad933d3ee5646b0e78f8/yes-no.el";
```

```suggestion
    url = "https://raw.githubusercontent.com/emacsmirror/emacswiki.org/143bcaeb679a8fa8a548e92a5a9d5c2baff50d9c/yes-no.el";
```

This is the most recent version of this file.

---

Updated! Thanks!